frontend/src/components/layout/HeaderAuthorized.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import VerifiedIcon from '@mui/icons-material/Verified';
+
+import PATHS from '../../constants/routes';
+import {
+  LayoutHeader, 
+  HeaderDivider, 
+  SpaceHolder,
+  BrandLogo, 
+  HeaderContent, 
+  MyCoursesCounter, 
+  MyCoursesCounterContent, 
+  MyCoursesCounterNumber, 
+  LogOut
+} from './styled'
+import UserDropDownContainer from './UserDropDownContainer'
+
+interface Props {
+  user?: {
+    role?: string,
+    unit?: string,
+    department?: string,
+    group?: string
+  };
+  myCourses?: number;
+  avatarUrl?: string;
+  userName?: string;
+}
+
+const HeaderAuthorized: React.FC<Props> = ({ user, myCourses, avatarUrl, userName }) => (
+  <LayoutHeader>
+    <NavLink to={PATHS.home}>
+      <BrandLogo>
+        Skill Up Level
+      </BrandLogo>
+    </NavLink>
+    <HeaderContent>
+      <SpaceHolder/>
+      <MyCoursesCounter>
+        <MyCoursesCounterContent>
+          <VerifiedIcon fontSize='medium' color='secondary'/>
+          <MyCoursesCounterNumber>
+            {myCourses}
+          </MyCoursesCounterNumber>
+        </MyCoursesCounterContent>
+      </MyCoursesCounter>
+      <HeaderDivider />
+        <UserDropDownContainer user={user} avatarUrl={avatarUrl} userName={userName} />
```

Comment:
Remove please Container from the name. There is no need to know here is container or component.
Example:` import UserDropDownContainer from './UserDropDown'`
And also don't forget to use `index.tsx` file inside `UserDropDown` folder